lively.graphics/geometry-2d.js

```diff
@@ -582,6 +582,10 @@ export class Rectangle {
       this.x + (r.x * this.width), this.y + (r.y * this.height), r.width * this.width, r.height * this.height);
   }
 
+  scaleByFactor (num) {
+    return new Rectangle(this.x, this.y, this.width * num, this.height * num);
+  }
+
```

Comment:
I would prefer something that is more similar and capable to the function found in `Point`:
```suggestion
  scaleBy (scaleX, scaleYOrUndefined) {
    return new Rectangle(this.x, this.y, this.width * scaleX, this.height * (scaleYOrUndefined || scaleX));
  }

```